package.json

```diff
@@ -18,6 +18,7 @@
     "release:version": "lerna version --no-push --yes --no-private",
     "release:publish": "lerna publish from-package --no-private",
     "release": "npm run release:version && npm run release:build && npm run release:publish",
+    "release:connect:js": "./deploy_connect_web_js.sh",
```

Comment:
naming: release:connect:js  => release:connect-web-js? (the package is called connect-web-js)